### PR TITLE
[Bugfix] Add detect for bitmap function

### PR DESCRIPTION
**Pull request by Pslydhh**
## What type of PR is this：
- [x] bugfix
- [ ] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

When size of bitmap is large, we should end expansion function in advance, so add three config to detect in **to_base64(max_length_for_to_base64), bitmap_to_string(max_length_for_bitmap_to_string), bitmap_to_array(max_length_for_bitmap_to_array)**.

### CODE REVIEW

File: be/src/common/config.h [hunk 1]
```diff
@@ -818,4 +818,11 @@ CONF_Int32(cardinality_of_inject, "100");
  * Split by "," strictly.
  */
 CONF_String(directory_of_inject, "/src/exec,/src/exprs");
+
+// Used by to_base64
+CONF_Int64(max_length_for_to_base64, "200000");
+// Used by bitmap_to_string
+CONF_Int64(max_length_for_bitmap_to_string, "200000");
```
Comment (kangkaisen) [inline be/src/common/config.h:825]: bitmap_to_string and bitmap_to_array only need one config.  could be 1000000.

Comment (Pslydhh) [reply be/src/common/config.h]: done.

**Review by kangkaisen — DISMISSED** [review]

**Review by kangkaisen — APPROVED** [review]

**Review by satanson — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by satanson** [comment]
run starrocks_admit_test

**Comment by satanson** [comment]
run starrocks_admit_test

**Pull request merged**